homeassistant/components/plugwise/climate.py

```diff
@@ -124,15 +139,18 @@ def extra_state_attributes(self) -> Mapping[str, Any] | None:
     @plugwise_command
     async def async_set_temperature(self, **kwargs: Any) -> None:
         """Set new target temperature."""
-        if ((temperature := kwargs.get(ATTR_TEMPERATURE)) is None) or (
-            self._attr_max_temp < temperature < self._attr_min_temp
+        if ((temperature := kwargs.get(ATTR_TEMPERATURE)) is None) or not (
+            self._attr_min_temp < temperature < self._attr_max_temp
         ):
             raise ValueError("Invalid temperature requested")
```

Comment:
I can change it to `HomeAssistantError("temperature value out-of-range")`?

Reply:
Yeah or just change `ValueError` to `HomeAssistantError`. It will still fail as normal, except it would allow an end-user to set `continue_on_error` if they want... maybe?

Interesting, never thought too much about this one from this perspective...

@MartinHjelmare Sorry for interrupting, should a user be able to skip/ignore an error like this in their automations? Or should it be a hard failure always? What is your opinion that?

Reply:
Right makes sense actually, let's keep it as is. Thanks Martin 👍